t/rose-macro/test_header

```diff
@@ -77,3 +77,15 @@ function teardown() {
     rm -rf $TEST_DIR/run
     rm -rf $TEST_DIR/config/meta
 }
+
+function grep_count_ok() {
```

Comment:
Done, though I was following the example of `t/lib/bash/test_header:file_grep()`.